plugins.md

```diff
@@ -0,0 +1,218 @@
+# Trace Plugin API Developer Guide
+
+**Note: The Trace Plugin API is experimental and is subject to change without notice.**
+
+The trace agent is driven by a set of plugins that describe how to patch a
+module to generate trace spans when that module is used. We provide plugins for
+some well-known modules such as express, mongodb, and http, and provide a plugin
+API for third-party developers to create their own.
```

Comment:
I would use a different word from "third-party" here. Maybe just: "for developers to create their own."